mysite/static/js/account/edit_photo.js

```diff
@@ -0,0 +1,64 @@
+var csrf_token = null;
+var width = null;
+
+function set_callback_for_image_upload(token){
+    $("#edit_photo").on("submit", upload_image);
+    csrf_token = token;
+    width = $("#progressBar").width();
+    $('#clear_message').click(hide_notification);
```

Comment:
This is an excellent point! 
I didn't notice how badly `.click()` read. Thanks for bringing this up.
I could take up testing this on different browsers; It works with Chrome(38.0.2125.111) and Firefox(33.0), but I could test it for newer versions of these browsers too.